testing/test_full_execution_checkpoint.py

```diff
@@ -0,0 +1,200 @@
+import os
+
+from joblib import dump
+
+from neuraxle.pipeline import ResumablePipeline, FullExecutionCheckpoint
+from neuraxle.steps.misc import TapeCallbackFunction, FitTransformCallbackStep
+from testing.test_checkpoints import CheckpointTest
+
+EXPECTED_OUTPUT_CHECKPOINTER_WRAPPER = 'ExpectedOutputCheckpointerWrapper'
+
+DATA_INPUT_CHECKPOINTER_WRAPPER = 'DataInputCheckpointerWrapper'
+
+FIT_TRANSFORM_ONLY_WRAPPER_DI_NAME = 'FitTransformOnlyWrapperDi'
+TRANSFORM_ONLY_WRAPPER_DI_NAME = 'TransformOnlyWrapperDi'
+FIT_ONLY_WRAPPER_DI_NAME = 'FitOnlyWrapperDi'
+
+FIT_TRANSFORM_ONLY_WRAPPER_EO_NAME = 'FitTransformOnlyWrapperEo'
+TRANSFORM_ONLY_WRAPPER_EO_NAME = 'TransformOnlyWrapperEo'
+FIT_ONLY_WRAPPER_EO_NAME = 'FitOnlyWrapperEo'
```

Comment:
It looks like some tests of this file can be parametrized with lists of tuple of those constants.